tests/unit/test_config.py

```diff
@@ -1,13 +1,19 @@
 """Test logging in config.py."""
 
+import logging
+
 import pytest
 
 from wristpy.core import config
 
 
 def test_get_logger(caplog: pytest.LogCaptureFixture) -> None:
-    """Test the wristpy logger with level set to 20 (info)."""
-    logger = config.get_logger()
+    """Test the wristpy logger with level set to default 20 (info)."""
+    if logging.getLogger("wristpy").handlers:
```

Comment:
You can move `logger = ...` out of the if-statement and drop the else statement entirely.